### PR TITLE
feat(client-go): add releases endpoint

### PR DESCRIPTION
**Pull request by Joshua-Anderson**

### CODE REVIEW

**Comment by mboersma** [comment]
Code LGTM.

File: client-go/cmd/releases.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+	r, err := releases.Get(c, appID, version)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("=== %s Release v%d\n", appID, version)
+	if r.Build != "" {
+		fmt.Println("build:   ", r.Build)
```
Comment (bacongobbler) [inline client-go/cmd/releases.go]: I feel like these titles should be capitalized for formality

Comment (Joshua-Anderson) [inline client-go/cmd/releases.go]: Nothing else like this is capitalized, so it would be a client wide change. :smiley:

**Comment by bacongobbler** [comment]
code LGTM

**Comment by mboersma** [comment]
Needs a rebase now--sorry about that. :repeat_one:

**Comment by Joshua-Anderson** [comment]
Rebased!

**Pull request merged**